parsimonious/tests/test_nodes.py

```diff
@@ -145,6 +156,33 @@ def visit_greeting(self, thing, visited_children):
 
 
 def test_node_inequality():
-    node = Node('text', 'o hai', 0, 5)
+    node = Node(Literal('12345'), 'o hai', 0, 5)
     ok_(node != 5)
     ok_(node != None)
+    ok_(node != Node(Literal('23456'), 'o hai', 0, 5))
+    ok_(not (node != Node(Literal('12345'), 'o hai', 0, 5)))
+
+
+def test_generic_visit_NotImplementedError_unnamed_node():
+    class MyVisitor(NodeVisitor):
```

Comment:
Could we have docstrings for these 2 new tests so I can tell at a glance what behavior we're testing?